pkg/utils/validator/clusterresourceplacement.go

```diff
@@ -91,9 +92,10 @@ func ValidateClusterResourcePlacement(clusterResourcePlacement *placementv1beta1
 			allErr = append(allErr, fmt.Errorf("the clusterAffinity field is invalid: %w", err))
 		}
 	}
-
-	if err := validateRolloutStrategy(clusterResourcePlacement.Spec.Strategy); err != nil {
-		allErr = append(allErr, fmt.Errorf("the rollout Strategy field  is invalid: %w", err))
+	if !reflect.DeepEqual(clusterResourcePlacement.Spec.Strategy, placementv1beta1.RolloutStrategy{}) {
```

Comment:
why compare it with an empty struct?

Reply:
I'm comparing it to an empty object because rollout strategy is not a pointer, not this check if being called before calling validateRolloutStrategy